web/requirements.in

```diff
@@ -1,4 +1,4 @@
-cpg-utils==4.18.3
+cpg-utils~=5.0.1
```

Comment:
so this will pick up the latest in the 5.0.X range? Below we have `cpg-utils==5.0.10`, so should this be `cpg-utils~=5.0.10`

Reply:
should have refreshed the page eh

Reply:
It's effectively the same thing, the requirements.in is just saying this should work with anything above `5.0.1` but less than `5.1`, in some ways it is nice to keep it kind of loose, because if another dependency were to specify `cpg-utils==5.0.4`, `pip-compile` could resolve the conflict and decide to use `5.0.4`. Whereas if we specified `~=5.0.10` here that wouldn't be a resolvable conflict